dpc-sdp/ripple-framework
# Feature/map clustering distance prop

**Issue**:
PR to accompany feature request https://github.com/dpc-sdp/ripple-framework/issues/1290

### What I did

- Added optional `clusteringDistance` prop to `RplMap` & passed it to `ol-animated-clusterlayer` in the template
- Added Map 'Clustering Nearer' & 'Clustering Disabled' stories to Storybook

### How to test

- See the new stories in the Storybook and confirm clustering behaviour has changed

### Checklist




#### For all PR's


#### For new components only
